client/layout/masterbar/logged-in.jsx

```diff
@@ -68,12 +73,26 @@ class MasterbarLoggedIn extends React.Component {
 		}
 	};
 
+	actionSearchShortcut( ctx ) {
```

Comment:
What does ctx mean here? Is a clearer name possible?

Reply:
Yeah I see a problem with this approach, fixing this.